SEGSupport/EncodeSEG.cxx

```diff
@@ -38,12 +38,12 @@
 #include <itkBinaryThresholdImageFilter.h>
 
 // UIDs
-#include "../Common/QIICRUIDs.h"
-#include "../Common/SegmentAttributes.h"
+#include "QIICRUIDs.h"
+#include "SegmentAttributes.h"
 //#include "../Common/conditionCheckMacros.h"
 
 // versioning
-#include "../Iowa2DICOMVersionConfigure.h"
+#include "../vtkReportingVersionConfigure.h"
```

Comment:
Would be nice to avoid relative important. If needed, the directory should be added as `INCLUDE_DIRECTORIES` when calling `SEMMacroBuildCLI`